MLK_Uninstall/HelperClass.cs

```diff
@@ -1,37 +1,33 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Text;
-using System.Net;
 using System.IO;
-using System.Threading;
-using System.Security.Permissions;
-using System.Security.AccessControl;
 using System.Security.Principal;
 using System.Windows.Forms;
-using Microsoft.VisualBasic;
 using Microsoft.Win32;
 using System.Runtime.InteropServices;
-using System.Runtime.InteropServices.ComTypes;
 
 namespace MLK_Uninstall
 {
     class HelperClass
     {
+        [DllImport("shell32.dll")]
```

Comment:
it is usually considered to be cleaner to have Dll Imports close to the code where they are actually used instead of at class head. maybe consider moving it back?